__tests__/Hal.spec.js

```diff
@@ -0,0 +1,159 @@
+
+
+describe('Hal', () => {
+  let test;
+
+  beforeEach(() => {
+    test = {};
+    test.Hal = require('../lib/Hal')
+  })
+
+  describe('constructor', () => {
+    beforeEach(() => {
+      test.phrase = 'Hello'
+      test.subject = new test.Hal(test.phrase);
```

Comment:
Nice work setting up tests in `beforeEach`, it helps keep you `it` blocks clean.